src/compements/Form.jsx

```diff
@@ -0,0 +1,65 @@
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+import '../styles/Form.css'
+// eslint-disable-next-line react/prop-types
+function Form({firstname,setFirstname, lastname, setLastname, age, setAge, postcode, setPostcode, deliveryType, setDeliveryType, selectedOption,setSelectedOption, message, setMessage,terms,setTerms}) {
```

Comment:
Je hebt een component gemaakt maar hij is niet herbruikbaar. In de antwoorden op github staat een voorbeeld hoe je het herbruikbaar zou kunnen maken :)